ReadMe.pod

```diff
@@ -137,7 +137,13 @@ Add a repository as a subrepo in a subdir of your repository.
 
 This is similar in feel to C<git clone>. You just specify the remote repo url,
 and optionally a sub-directory and/or branch name. The repo will be fetched
-and merged into the subdir.
+and merged into the subdir. When you clone a subrepo:
+
+=item * The current working directory must be at the top of the parent repository.
```

Comment:
I believe this would be better stated that git subrepo command must be run from the root of the git repo.

Reply:
I agree.